frontera/contrib/backends/remote/codecs/msgpack.py

```diff
@@ -80,18 +82,18 @@ def decode(self, buffer):
         if obj[0] == 'pc':
             return ('page_crawled',
                     self._response_from_object(obj[1]),
-                    map(self._request_from_object, obj[2]))
+                    list(map(self._request_from_object, obj[2])))
```

Comment:
Agreed privately, that Preet will check all cases where list is used, and remove those where it's not needed.